lib/registration.pm

```diff
@@ -262,8 +262,15 @@ sub registration_bootloader_params {
 }
 
 sub yast_scc_registration {
-
-    type_string "yast2 scc; echo yast-scc-done-\$?- > /dev/$serialdev\n";
+    my ($type_slow) = @_;
+    my $yast_scc_cmd = "yast2 scc; echo yast-scc-done-\$?- > /dev/$serialdev\n";
+    #Due to ninja keys on PPC, may be required to type command slowlier, see poo#19230
+    if ($type_slow) {
```

Comment:
Agreed, changing VNC_TYPING_LIMIT is better idea as may help with other test failures like one here: https://progress.opensuse.org/issues/19266 https://progress.opensuse.org/issues/19414 which never happened on malbec.